app/models/businessTaxReliefs/OtherReliefs.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.businessTaxReliefs
+
+import play.api.libs.json.{Json, OFormat}
+
+case class OtherReliefs(qualifyingLoanInterestPayments: Seq[QualifyingLoanInterestPayments])
```

Comment:
We also need to add the two other models here for:
- nonDeductableLoanInterest
- postCessationTradeReliefAndCertainOtherLosses

Both of these should be added as `Option[...]` too.

Reply:
👍🏽 I'll add that as part of the other PR with the service layer